trainingMaquetado/maquetado.css

```diff
@@ -6,6 +6,30 @@
   margin-bottom: 20px;
 }
 
+.subscribe {
+  background-color: Transparent;
+  border-color: white;
```

Comment:
use hex notation for colors to be consistent, #FFF for this particular case. also the border can be written like this: `border: 1px solid #FFF;`